### PR TITLE
Release v1.19.0

### PR DESCRIPTION
**Pull request by angrybayblade**
## Release summary

Version number: 1.19.0

## Release details

AEA:
- Updates the `aea init` command to set the local as default registry and IPFS as default remote registry
- Updates the `aea test packages` to include the agent tests
- Introduces 
  - `aea packages` command group to manage local packages repository
  - `aea packages lock` command to lock all available packages and create `packages.json` file 
  - `aea packages sync` command to synchronize the local packages repository

Chores:
- Fix README header link
- Removes `shebangs` from non-script files
- Adds a command validator for docs and Makefile
- Deprecates the usage of `hashes.csv` to maintain packages consistency

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side), from `develop`
- [x] Lint and unit tests pass locally and in CI
- [x] I have checked the fingerprint hashes are correct by running (`aea hash all` and `aea packages lock --check`)
- [x] I have regenerated the latest API docs
- [x] I built the documentation and updated it with the latest changes
- [x] I have added an item in `HISTORY.md` for this release
- [x] I bumped the version number in the `aea/__version__.py` file.
- [ ] I bumped the version number in every Docker image of the repo and published it. Also, I built and published them with tag `latest`  
      (check the READMEs of [`aea-develop`](../develop-image/README.md#publish) 
      and [`aea-user`](../user-image/README.md#publish))
- [ ] I have pushed the latest packages to the registry.
- [ ] I have uploaded the latest `aea` to PyPI.
- [ ] I have uploaded the latest plugins to PyPI.

### CODE REVIEW

File: docs/upgrading.md [hunk 1]
```diff
@@ -6,6 +6,15 @@ Below we describe the additional manual steps required to upgrade between differ
 
 
 # Open AEA
+
+## `v1.18.0` to `v1.19.0`
+
+- Deprecated the usage of `hashes.csv` and replaces it with `packages.json`
```
Comment (DavidMinarsch) [inline docs/upgrading.md:12]: Explain what command the user needs to run in a second sentence.

File: docs/upgrading.md [hunk 2]
```diff
@@ -6,6 +6,15 @@ Below we describe the additional manual steps required to upgrade between differ
...
+- `--check` flag is deprecated from `aea hash all`
```
Comment (DavidMinarsch) [inline docs/upgrading.md:13]: Explain what replaces this in a second sentence.

File: tox.ini [hunk 3]
```diff
@@ -380,6 +380,10 @@ deps =
     pycryptodome>=3.10.1
     cosmpy>=0.4.1,<0.5.0
     setuptools==59.6.0
+    certifi<=2022.6.15.1
+    google-api-python-client<=2.60.0
+    idna<=3.3
+    protobuf<=3.20.1
```
Comment (Karrenbelt) [inline tox.ini:386]: first 3 not in Pipfile and last one doesn't match specification there
https://github.com/valory-xyz/open-aea/blob/b1b9afbc3a6c97e10a8ddab7e0a27af24fc056e3/Pipfile#L48

File: HISTORY.md [hunk 4]
```diff
@@ -1,5 +1,21 @@
 # Release History - open AEA
 
+## 1.19.0 (2022-09-14)
+
```
Comment (DavidMinarsch) [inline HISTORY.md:4]: this could be added: https://github.com/valory-xyz/open-aea/pull/298

**Review by DavidMinarsch — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**